src/main/java/uk/gov/hmcts/cmc/claimstore/services/staff/content/SealedClaimContentProvider.java

```diff
@@ -20,32 +21,40 @@
     private final ClaimContentProvider claimContentProvider;
 
     @Autowired
-    public SealedClaimContentProvider(
-        ClaimantContentProvider claimantContentProvider,
-        PersonContentProvider personContentProvider, ClaimContentProvider claimContentProvider) {
+    public SealedClaimContentProvider(ClaimantContentProvider claimantContentProvider,
```

Comment:
why final is missed here??

Reply:
People have a tend to forget

Reply:
huh? If its part of our codestyle it should be part of checkstyle...
Its not something that we've committed to though.